### PR TITLE
New sensor integration for PSE&G utility gas meter

### PR DESCRIPTION
**Pull request by bvlaicu**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#https://github.com/home-assistant/home-assistant/pull/27902

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/pseg.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+{% configuration %}
+energize_id:
+  description: The energize id corresponding to your PSE&G customer account. 
+  required: true
+  type: string
+session_id:
+  description: The session id corresponding to your PSE&G customer account. 
+  required: true
```
Comment (frenck) [inline source/_integrations/pseg.markdown:31]: According to the code base, this is optional?

<https://github.com/home-assistant/home-assistant/pull/27902/files#diff-3464f58a5d9ec8e77ebe85c9a972e414R32>

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there, @bvlaicu! Thanks for you PR!

I've reviewed it and left a comment, could you please take a look? Thanks! 👍

File: source/_integrations/pseg.markdown (hunk 1 above)
Comment (bvlaicu) [reply source/_integrations/pseg.markdown:31]: Nice catch, @frenck. I fixed the code base to mark it as Required. Thanks!

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_integrations/pseg.markdown [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# Example configuration.yaml entry
+sensor:
+  - platform: pseg
+    energize_id: YOUR_ENERGIZE_ID
+    session_id: YOUR_SESSION_ID
+```
+
+{% configuration %}
+energize_id:
+  description: The energize id corresponding to your PSE&G customer account.
```
Comment (fabaff) [inline source/_integrations/pseg.markdown:26]: Hmm, at the button you are writing that it's a cookie parameter.

File: source/_integrations/pseg.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+session_id:
+  description: The session id corresponding to your PSE&G customer account. 
+  required: true
+  type: string
+{% endconfiguration %}
+
+Both `energize_id` and `session_id` can be found as browser cookies after logging in into your PSE&G customer portal at [myenergy.pseg.com](https://myenergy.pseg.com/energy_use)
+Just log in, then put the following command in the browser address and hit enter: `javascript: alert(document.cookie)`.
+The `energize_id` parameter is the value of the `_energize_session` cookie.
+The `session_id` parameter is the value of the `EMSSESSIONID` cookie.
```
Comment (fabaff) [inline source/_integrations/pseg.markdown:38]: This should go into a "setup" section before the "configuration" section.

**Review by fabaff — CHANGES_REQUESTED** [review]

**Pull request closed without merging**